client/src/App.js

```diff
@@ -29,28 +51,35 @@ export function App() {
 			</div>
 			<Router>
 				<nav className="col-4">
-					<ul><li style={{ listStyleType: "none" }}>
-						<Link to="/NewQuestion" >New question</Link>
-					</li>
+					<ul>
 		  <li style={{ listStyleType: "none" }}>
-						<Link to="/Students" exact='true'>for students </Link>
-					</li>
-					<li style={{ listStyleType: "none" }}>
-						<Link to="/Mentors"exact='true'> for mentors</Link>
-					</li>
-
+							<Link to="/Students" exact='true'>student </Link>
+						</li>
+						<li style={{ listStyleType: "none" }}>
+							<Link to="/Mentors"exact='true'> mentor</Link>
+						</li>
+						<li style={{ listStyleType: "none" }}>
+							<Link to="/Results"exact='true'> quiz rezults</Link>
```

Comment:
`quiz results`

Also you're missing a space before `exact` in the `Link` (same above with mentors)